src/NumericalAlgorithms/LinearOperators/Python/PowerMonitors.cpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "NumericalAlgorithms/LinearOperators/Python/PowerMonitors.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "DataStructures/DataVector.hpp"
+#include "NumericalAlgorithms/LinearOperators/PowerMonitors.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+
+namespace py = pybind11;
+
+namespace PowerMonitors::py_bindings {
+
+void bind_power_monitor_array(py::module& m) {
+  m.def(
+      "power_monitor_array",
+      [](const DataVector& input_data_vector, const Mesh<1>& mesh) {
+        return PowerMonitors::power_monitor_array(input_data_vector, mesh);
+      },
```

Comment:
You don't need to wrap this in a lambda function. Just write `&power_monitors`. When you have the two overloads, write `py::overload_cast<const DataVector&, const Mesh<Dim>&>(&power_monitors)`.